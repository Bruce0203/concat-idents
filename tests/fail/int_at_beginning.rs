concat_idents::concat_idents!(fn_name = 1, foo, bar, {
    fn fn_name() {
        /* integers are not allowed at the beginning of an ident */
    }
});

fn main() {}
