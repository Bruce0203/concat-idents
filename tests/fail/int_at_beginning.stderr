error: Identifies cannot start with integers
 --> $DIR/int_at_beginning.rs:1:41
  |
1 | concat_idents::concat_idents!(fn_name = 1, foo, bar, {
  |                                         ^
